Python 2.7.15 (v2.7.15:ca079a3ea3, Apr 30 2018, 16:30:26) [MSC v.1500 64 bit (AMD64)] on win32
Type "copyright", "credits" or "license()" for more information.
>>> T=((0,1)(1,4))

Traceback (most recent call last):
  File "<pyshell#0>", line 1, in <module>
    T=((0,1)(1,4))
TypeError: 'tuple' object is not callable
>>> T=((0,1),(1,4))
>>> T
((0, 1), (1, 4))
>>> T[0]
(0, 1)
>>> T[0]=(1,3)

Traceback (most recent call last):
  File "<pyshell#4>", line 1, in <module>
    T[0]=(1,3)
TypeError: 'tuple' object does not support item assignment
'
>>> T=([0,1],[1,4])
>>> T[0]=(1,3)

Traceback (most recent call last):
  File "<pyshell#6>", line 1, in <module>
    T[0]=(1,3)
TypeError: 'tuple' object does not support item assignment
>>> T[0][1]=4
>>> T
([0, 4], [1, 4])
>>> T
([0, 4], [1, 4])
>>> list(T)
[[0, 4], [1, 4]]
>>> tuple(T)
([0, 4], [1, 4])
>>> 
